pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -206,6 +213,8 @@ def training_type_plugin(self) -> TrainingTypePlugin:
             self._training_type_plugin = self.select_training_type_plugin()
         else:
             self._training_type_plugin = self.resolve_training_type_plugin(self._training_type_plugin)
+        # attach local_rank
+        self._training_type_plugin.task_idx = self.local_rank
```

Comment:
I hope this is just for debugging sake, because this is a bit elusive

Reply:
Yes, and it is wrong. Let me move it in ddp_spawn